wfw/lib.py

```diff
@@ -26,7 +26,7 @@ def get_user_data():
             configparser.readfp(config)
         user_data['email'] = configparser.get('user', 'email')
         user_data['password'] = configparser.get('user', 'password')
-        user_data['aux'] = configparser.get('tree', 'shared')
+        user_data['shared'] = configparser.get('tree', 'shared')
```

Comment:
It should be `configparser.getboolean`, because now the value is string, so checking if `self.shared` is `True` always evaluates to true. My bad.